test/FunctionalTests/Microsoft.OData.Core.Tests/UriParser/ODataUriParserTests.cs

```diff
@@ -677,7 +677,152 @@ public void ParsePathFunctionWithNullEntitySetPath()
             pathSegmentList[0].ShouldBeOperationImportSegment(getCurrentCustomerImport);
             pathSegmentList[1].ShouldBeNavigationPropertySegment(customerDetail);
         }
+        #endregion
 
+        #region Parse Compute Tests

```

Comment:
I'll add something for 'as', but the remainder should be covered by the expression parsing reuse.

---
In reply to: [123090919](https://github.com/OData/odata.net/pull/827#discussion_r123090919) [](ancestors = 123090919)